channels.json5

```diff
@@ -514,4 +514,39 @@ Add another row at the end of the list, similar to the following object
     "name": "Nasser Ghanemzadeh",
     "category": "misc",
   },
+  {
+    "id": "UC9L_sf_AUqBuJi1cgCUC1Rw",
+    "name": "Hadi Moshayedi",
+    "category": "computer",
+  },
+  {
+    "id": "UC8q-CCyGP20I78UQFCovDCw",
+    "name": "Dev Quest",
+    "category": "computer",
+  },
+  {
+    "id": "UCvRY5AQuQy91hUhpRbJayvg",
+    "name": "Bazikhar Videocast",
+    "category": "misc",
+  },
+  {
+    "id": "UCdtGLDEOsnWlW1IsQFbCAsA",
```

Comment:
It's duplicated.
You can replace it with my channel :D

Reply:
OK :]